adapters/repos/db/index.go

```diff
@@ -2002,7 +2002,13 @@ func (i *Index) dropCloudShards(ctx context.Context, cloud modulecapabilities.Of
 			i.shardCreateLocks.Lock(name)
 			defer i.shardCreateLocks.Unlock(name)
 
-			return cloud.Delete(ctx, i.ID(), name, nodeId)
+			err := cloud.Delete(ctx, i.ID(), name, nodeId)
+			if err != nil {
+				ec.Add(err)
+				i.logger.WithField("action", "cloud_drop_shard").
+					WithField("shard", name).Error(err)
+			}
+			return err
```

Comment:
you are adding this error to `ec.Add(err)` but returning `err` is that ok?

Reply:
shouldn't be there `return ec.ToError()`?

Reply:
yeah return the error os for eg and ec for the whole function

Reply:
actually it should be nil, so that eg will continue